src/main/java/core/basesyntax/Lottery.java

```diff
@@ -0,0 +1,9 @@
+package core.basesyntax;
+
+public class Lottery {
+    public static Ball getRandomBall() {
+        String color = ColorSupplier.getRandomColor();
+        int number = (int) (Math.random() * 100) + 1;
```

Comment:
Random and ColorSupplier must be a class field
use constants instead of numbers
[this article](https://mate-academy.github.io/style-guides/java/java.html#s5.2.4-constant-names)